src/raftstore/store/peer.rs

```diff
@@ -464,6 +464,35 @@ impl Peer {
         self.raft_group.raft.state == StateRole::Leader
     }
 
+    // When this peer is the leader of the region before split,
+    // `accelerate_campaign_ticks` specifies the tick number to be accelerated
+    // after the region split. So this peer of new split region may campaign and become leader
+    // earlier than other follower peers.
+    pub fn accelerate_campaign_ticks(&self) -> usize {
+        // The best case for the fast campaign after a split is:
+        //   1. The leader of old region commits the log entry which contains the corresponding
```

Comment:
Two indexes involved here.
0. leader send Append Entries to tell followers to append this split log.
1. leader commit.
2. leader send Append Entries to tell follower to commit this split log.